heapster-saw/src/Verifier/SAW/Heapster/TypedCrucible.hs

```diff
@@ -1526,7 +1557,7 @@ deleteEntryCallees topEntryID = execState (deleteCallees topEntryID) where
                  . entryByID calleeID) $ \(Some callee) ->
       let callers' =
             flip filter (typedEntryCallers callee) $ \(Some site) ->
-            callSiteIDCallerEq callerID $ typedCallSiteID site in
+            not $ callSiteIDCallerEq callerID $ typedCallSiteID site in
```

Comment:
that's indeed a significant difference

Reply:
Yeah, seriously. This one line change is the main reason for this entire PR. :/